src/DataStructures/LeviCivitaIterator.cpp

```diff
@@ -3,13 +3,6 @@
 
 #include "DataStructures/LeviCivitaIterator.hpp"
 
```

Comment:
shouldn't this be with the other commit touching the LeviCivitaIterator?

Reply:
I don't see your comment attached to any particular code, so I'm not sure what you're referring to.

If you're speaking of the clang-tidy fix, I originally had it separate to reduce the non-find-and-replace operations in the renaming commit, but if you want me to squash it in, I'm happy to do so.

Reply:
never mind...